### PR TITLE
Removed country tooltips, closes getlantern/lantern#2265

### PR DESCRIPTION
**Pull request by oxtoacart**

### CODE REVIEW

File: src/github.com/getlantern/lantern-ui/app/js/vis.js [hunk 1]
```diff
@@ -97,12 +97,12 @@ angular.module('app.vis', ['ngSanitize'])
...
             el.attr('class', 'COUNTRY_KNOWN');
             if (d.alpha2) {
-              var $content = ttTmpl(d.alpha2);
+              //var $content = ttTmpl(d.alpha2);
 
               el.attr('class', d.alpha2 + " COUNTRY_KNOWN")
-                .attr('tooltip-placement', 'mouse')
-                .attr('tooltip-html-unsafe', $content);
-                $compile(this)(scope);
+                .attr('tooltip-placement', 'mouse');
```
Comment (atavism) [inline src/github.com/getlantern/lantern-ui/app/js/vis.js]: You should be able to comment this attribute out as well

Comment (oxtoacart) [inline src/github.com/getlantern/lantern-ui/app/js/vis.js]: Good point!  Done.

**Comment by atavism** [comment]
Ok, LGTM.

**Comment by oxtoacart** [comment]
Thanks @atavism

**Pull request merged**